src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy

```diff
@@ -103,7 +104,7 @@ class IqPolicyEvaluatorIntegrationTest
     when: 'the nexus policy evaluator is executed'
       project.definition = new CpsFlowDefinition('node {\n' +
           'writeFile file: \'dummy.txt\', text: \'dummy\'\n' +
-          'def result = nexusPolicyEvaluation failBuildOnNetworkError: false, iqApplication: \'app\', ' +
+          'def result = nexusPolicyEvaluation failBuildOnNetworkError: false,  iqApplication: \'select\', ' +
```

Comment:
Weird formatting here, above and below with double spaces.

Reply:
> Weird formatting here, above and below with double spaces.

Changed 90ed6e3

Reply:
Missed some